pkgs/top-level/all-packages.nix

```diff
@@ -40630,4 +40630,6 @@ with pkgs;
   isolate = callPackage ../tools/security/isolate { };
 
   reindeer = callPackage ../development/tools/reindeer { };
+
+  questdb = callPackage ../servers/nosql/questdb { };
```

Comment:
Please sort this into a fitting category and sort there alphabetically.

Reply:
I have moved it, hopefully to the right place (but not sure, the file is pretty large and little bit messy)